### PR TITLE
Remove TimeDelta warnings in downtimes

### PR DESCRIPTION
**Pull request by rhiannonlynne**
Remove a mildly annoying warning message.

### CODE REVIEW

File: rubin_sim/site_models/scheduled_downtime_data.py [hunk 1]
```diff
@@ -82,12 +81,12 @@ def read_data(self):
         database. However, an alternate database file can be provided. The alternate
         database file needs to have a table called *Downtime* with the following columns:
 
-        night
-            int : The night (from start of simulation) the downtime occurs.
-        duration
-            int : The duration (units=days) of the downtime.
-        activity
-            str : A description of the activity involved.
+        night: `int`
```
Comment (tribeiro) [inline rubin_sim/site_models/scheduled_downtime_data.py:84]: Can you add a space between "night" and ":", e.g., 

```
night : `int`
```

File: rubin_sim/site_models/scheduled_downtime_data.py [hunk 2]
```diff
@@ -82,12 +81,12 @@ def read_data(self):
...
+            The night (from start of simulation) the downtime occurs.
+        duration : `int`
+            The duration (units=days) of the downtime.
+        activity: `str`
```
Comment (tribeiro) [inline rubin_sim/site_models/scheduled_downtime_data.py:88]: Space between "activity" and ":";

```
activity : `str`
```

**Review by tribeiro — APPROVED** [review]
Looks good.. I left 2 really minor comments about the docstring...

File: rubin_sim/site_models/scheduled_downtime_data.py (hunk 1 above)
Comment (rhiannonlynne) [reply rubin_sim/site_models/scheduled_downtime_data.py:84]: Yes, definitely - forgot the space.

Comment (tribeiro) [reply rubin_sim/site_models/scheduled_downtime_data.py]: Ah! you are absolutely right! So, yeah, I don't know..

Comment (rhiannonlynne) [reply rubin_sim/site_models/scheduled_downtime_data.py]: Looks like it works, although maybe not the clearest thing ever (a markdown table would likely be better, but it's not obvious how to accomplish this). Fine for now.

**Pull request merged**